src/uct/ib/base/ib_device.c

```diff
@@ -549,12 +552,135 @@ ucs_status_t uct_ib_device_query(uct_ib_device_t *dev,
     return UCS_OK;
 }
 
+#if defined(__x86_64__)
+static int uct_ib_device_cleanup_proc(void* arg) {
+    uct_ib_device_nb_close_ctx *ctx = arg;
+    static const char *process_name = "ucx_cleanup";
+    struct rlimit nofile;
+    int ret = -EINTR;
+    int fd;
+    char c;
+
+    prctl(PR_SET_NAME, (uintptr_t)&process_name, 0, 0, 0);
+
+    getrlimit(RLIMIT_NOFILE, &nofile);
```

Comment:
1. pls pass it in ctx to avoid glibc
2. add comment that glibc should be avoided